src/api/spec/services/workflows/yaml_downloader_spec.rb

```diff
@@ -11,17 +11,32 @@
     end
 
     context 'github' do
-      let(:payload) { { scm: 'github', target_branch: 'master', target_repository_full_name: 'openSUSE/obs-server' } }
+      let(:payload) { github_payload }
       let(:url) { "https://raw.githubusercontent.com/#{payload[:target_repository_full_name]}/#{payload[:target_branch]}/.obs/workflows.yml" }
 
       it { expect(Down).to have_received(:download).with(url, max_size: max_size) }
     end
 
     context 'gitlab' do
-      let(:payload) { { scm: 'gitlab', target_branch: 'master', path_with_namespace: 'openSUSE/obs-server' } }
+      let(:payload) { { scm: 'gitlab', target_branch: 'master', path_with_namespace: 'openSUSE/obs-server', scheme: 'https', host: 'gitlab.com' } }
       let(:url) { "https://gitlab.com/#{payload[:path_with_namespace]}/-/raw/#{payload[:target_branch]}/.obs/workflows.yml" }
 
       it { expect(Down).to have_received(:download).with(url, max_size: max_size) }
     end
   end
+
+  def github_payload
```

Comment:
Does this really need to be a method? `let(:github_payload)` should also be fine here.

Reply:
Sure, it's updated.